model_training/tekton/tasks/s3/aws-s3-load.yaml

```diff
@@ -23,7 +23,7 @@ spec:
     optional: true
   steps:
     - name: load
-      image: quay.io/sustainability/kepler_model_server/s3:v0.7-py3.10
+      image: quay.io/sustainable_computing_io/kepler_model_server/s3:latest
```

Comment:
I think we should add new input for the s3 image. 
For the PR, we can use a temporarily built image for test. applied this comment to other yaml.